apps/n8n/1.72.1/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   homeassistant:
-    image: n8nio/n8n:1.72.1
+    image: n8nio/n8n:1.73.1
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
The primary difference between these two lines is the image version from `n8nio/n8n:1.72.1` to `n8nio/n8n:1.73.1`. This change may introduce new features, bug fixes, or security updates available in version 1.73. However, without additional context, it's unclear whether this update requires adjustments elsewhere in your Docker Compose setup or configuration. If you have not customized any settings that depend on specific versions of N8N, it should work smoothly with minor changes due to compatibility upgrades in newer versions.

For further considerations:

- **Customizations**: Ensure that no custom configurations (environment variables, volumes) rely specifically on older N8N behaviors.
  
- **Logs and Configuration Changes**: Check if there were significant changes in logging output or configuration options in version 1.73 for your use case.

If you encounter issues after updating, review any relevant N8N documentation or community forums for compatibility details or known issues related to migration.

**Recommendation**:
Update only based on the necessity of using a more recent release unless you've thoroughly tested how it interacts with other parts of your environment.